solr/core/src/java/org/apache/solr/handler/admin/LoggingHandler.java

```diff
@@ -151,6 +156,9 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
       rsp.add("loggers", info);
     }
     rsp.setHttpCaching(false);
+    if (cc != null && AdminHandlersProxy.maybeProxyToNodes(req, rsp, cc)) {
```

Comment:
Why do this at the end of this method instead of at the top (along with disabling http caching)?  By doing it at the end, if nodes=FOO yet the current node is BAR, it would do work it shouldn't be doing; right?  Notice where the other callers of AdminHandlersProxy.maybeProxyToNodes put their logic.

Reply:
SystemInfoHandler is doing similar to this logic;

Reply:
SIH is doing what I suggest, which is different than what the PR is doing.